config/index.js

```diff
@@ -11,6 +11,7 @@ var datadir = process.env.DATADIR || (
     ? path.join(__dirname, '..') : os.homedir())
 
 var config = {
+  archiver: '/tmp',
```

Comment:
This is in shared.archiver, I don't think that anything at the top level is used (and if it is, it shouldn't be)